sdk/identity/azure-identity/CHANGELOG.md

```diff
@@ -1,10 +1,20 @@
 # Release History
 
+## 1.6.0-beta.1 (Unreleased)
+
+### Features Added
+
+### Breaking Changes
+
+### Bugs Fixed
+
+### Other Changes
+
 ## 1.5.1 (2023-07-06)
 
 ### Bugs Fixed
 
-- [#4723]](https://github.com/Azure/azure-sdk-for-cpp/issues/4723) Accept a wider variety of token responses.
+- [#4723](https://github.com/Azure/azure-sdk-for-cpp/issues/4723) Accept a wider variety of token responses.
```

Comment:
To be consistent with others below, add the missing brackets, rather than removing one.

```suggestion
- [[#4723]](https://github.com/Azure/azure-sdk-for-cpp/issues/4723) Accept a wider variety of token responses.
```